src/python_bindings/data/bind_data_types.cpp

```diff
@@ -23,6 +34,11 @@ void BindDataTypes(py::module_& main_module) {
     using namespace model;
     py::class_<ColumnCombination>(data_module, "ColumnCombination")
             .def("__str__", &ColumnCombination::ToString)
+            .def("to_index_tuple",
+                 [](ColumnCombination const& cc) {
+                     return py::make_tuple(cc.GetTableIndex(),
+                                           VectorToTuple(cc.GetColumnIndices()));
+                 })
```

Comment:
As far as I know, `py::cast()` can convert a `std::vector` directly to a python tuple without the need to manually iterate over the elements.

Reply:
```cpp
return py::make_tuple(cc.GetTableIndex(), py::cast(cc.GetColumnIndices()));
```